schemas/iso19139/src/main/plugin/iso19139/layout/layout-custom-fields-keywords.xsl

```diff
@@ -84,6 +84,13 @@
           <xsl:value-of select="$thesaurusTitleEl/gmd:PT_FreeText/gmd:textGroup/
                                   gmd:LocalisedCharacterString[normalize-space(text()) != ''][1]"/>
         </xsl:when>
+        <xsl:when test="normalize-space($thesaurusTitleEl/gmx:Anchor) != ''">
```

Comment:
This was also done in https://github.com/geonetwork/core-geonetwork/pull/3399/files#diff-6773b3b0f1b882a0f274de7c8272e420 with less duplication.